src/vector/_methods.py

```diff
@@ -781,11 +781,14 @@ def transform2D(self: SameVectorType, obj: TransformProtocol2D) -> SameVectorTyp
         """
         Arbitrarily transforms the vector(s) by
 
+        .. blacken-docs:off
```

Comment:
To maintain consistency in the `_methods.py` file, I have used `.. blacken-docs:off/on` here instead of replacing `python` with `pycon`. LMKWYT.